package-lock.json

```diff
@@ -4578,12 +4578,6 @@
                 "@babel/core": "^7.4.0-0"
             }
         },
-        "node_modules/@storybook/builder-webpack4/node_modules/@types/node": {
```

Comment:
I think you can revert this file as well

Reply:
this is because i ran `npm dedupe` after i uninstalled the unnecessary packages from before but your right these shouldn't be part of this PR 👍